vscode-dotnet-runtime-extension/README.md

```diff
@@ -4,6 +4,10 @@
 
 This extension provides a unified way for other extensions like the [C#] and [C# Dev Kit] extensions to install local, private versions of the .NET Runtime. This extension is not intended to be used directly by users to install .NET for development purposes because it only includes the .NET Runtime and not the entire .NET SDK.
 
+## Why do I have this extension?
+
+This extension was probably included as a dependency of the [C#] or [C# Dev Kit] extensions (though [Unity] and [.NET MAUI] extensions might also be the cause). These extensions call into this extension to provide a unified way of downloading per-extension copies of the .NET Runtime for those extensions to use internally. If you already have an installation of .NET that you'd like to use, see [the troubleshooting section below](#i-already-have-a-net-runtime-or-sdk-installed-and-i-want-to-use-it). If you want to remove this extension completely, you will need to uninstall any extensions that depend on it first. If this extension is uninstalled, any .NET Runtimes installed by it will also be removed.
```

Comment:
Although they are not first party, CMake also represents a very large subset of consumers, as well as the Bicep extension, which is first party.  We should mention these extensions.